src/Test.UnitTests.Sarif/Writers/SarifLoggerTests.cs

```diff
@@ -137,13 +137,13 @@ public void SarifLogger_RedactedCommandLine()
                 string commandLine = Environment.CommandLine;
                 string lowerCaseCommandLine = commandLine.ToLower();
 
-                if (lowerCaseCommandLine.Contains("testhost.") || lowerCaseCommandLine.Contains("\\xunit.console"))
+                if (lowerCaseCommandLine.Contains("testhost.") || lowerCaseCommandLine.Contains(Path.DirectorySeparatorChar + "xunit.console"))
                 {
-                    int index = commandLine.LastIndexOf("\\");
+                    int index = commandLine.LastIndexOf(Path.DirectorySeparatorChar);
                     string argumentToRedact = commandLine.Substring(0, index + 1);
                     tokensToRedact = new string[] { argumentToRedact };
                 }
-                else if (pathToExe.IndexOf(@"\Extensions", StringComparison.OrdinalIgnoreCase) != -1)
+                else if (pathToExe.IndexOf(Path.DirectorySeparatorChar + @"Extensions", StringComparison.OrdinalIgnoreCase) != -1)
```

Comment:
>@ [](http://example.com/codeflow?start=73&length=1)

remove '@' here and in the line 159.

Reply:
sounds good.

Reply:
I will also look and change other similar changes.

Reply:
fixed